packages/web-components/src/components/va-breadcrumbs/va-breadcrumbs.scss

```diff
@@ -39,7 +39,7 @@
   }
 
   .usa-breadcrumb__link:hover {
-    color: var(--color-primary-darker);
+    color: var(--color-primary-dark);
```

Comment:
Shifts up a level due to names and values being changed per [this comment](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2228#issuecomment-1836918069)